src/lib/utils/options/readers/tsconfig.ts

```diff
@@ -26,6 +26,30 @@ export class TSConfigReader extends OptionsComponent
     private static OPTIONS_KEY:string = 'tsconfig';
 
 
+    private readConfigFile(event:DiscoverEvent, fileName:string) {
+        if (!FS.existsSync(fileName)) {
```

Comment:
Do you know where we use the raw compiler options? My primary concern is that we may implement extends "wrong" if the compiler options ever change. If I've learnt anything from using the TypeScript compiler, it's that things can change at any time so implementing your own is usually bad.